src/components/ProjectList/listOfProjects.js

```diff
@@ -212,6 +212,13 @@ const projectList = [
     description: 'React Styleguidist is a component development environment with hot reloaded dev server and a living style guide that you can share with your team. It lists component propTypes and shows live, editable usage examples based on Markdown files.',
     tags: ['JavaScript', 'UI'],
   },
+  {
+    name: 'Flutter',
+    imageSrc: 'https://raw.githubusercontent.com/flutter/website/master/src/_assets/image/flutter-lockup.png',
+    projectLink: 'https://github.com/flutter/flutter/contribute',
+    description: 'Flutter makes it easy and fast to build beautiful apps for mobile and beyond.',
+    tags: ['Dart','Android','Flutter','UI'],
```

Comment:
Flutter is also for Web and iOS.

Reply:
Also, include the tag `Cross-platform` as it can be used for web/desktop apps as well.

Reply:
![Screenshot from 2020-05-31 01-05-08](https://user-images.githubusercontent.com/58987025/83330464-bc924a80-a2ac-11ea-8095-07654e95d5e4.png)